packages/expo-task-manager/src/TaskManager.ts

```diff
@@ -2,62 +2,109 @@ import { EventEmitter, UnavailabilityError } from '@unimodules/core';
 
 import ExpoTaskManager from './ExpoTaskManager';
 
-interface TaskError {
+/**
+ * Error object that can be received through TaskManagerTaskBody when the task fails.
+ */
+export interface TaskManagerError {
   code: string | number;
   message: string;
 }
 
-interface TaskBody {
+/**
+ * Represents the object that is passed to the task executor.
+ */
+export interface TaskManagerTaskBody {
+  /**
+   * An object of data passed to the task executor. Its properties depends on the type of the task.
+   */
   data: object;
-  error: TaskError | null;
+
+  /**
+   * Error object if the task failed or `null` otherwise.
+   */
+  error: TaskManagerError | null;
+
+  /**
+   * Additional details containing unique ID of task event and name of the task.
+   */
   executionInfo: {
     eventId: string;
     taskName: string;
   };
 }
 
-export interface RegisteredTask {
+/**
+ * An interface of already registered task.
```

Comment:
I don't think we need to write in a comment what's the semantics of given element 😉 
```suggestion
 * Represents an already registered task.
```